app/scripts/components/map/styles/mapbox-draw-styles.js

```diff
@@ -32,7 +32,7 @@ const styles = [
   {
     'id': 'gl-draw-line-inactive-edited',
     'type': 'line',
-    'filter': ['all', ['==', '$type', 'LineString'], ['!=', 'mode', 'static'], ['==', 'active', 'false'], ['==', 'user_status', 'edited']],
+    'filter': ['all', ['==', '$type', 'LineString'], ['!=', 'mode', 'static'], ['==', 'active', 'false'], ['==', 'user_status', 'in progress']],
```

Comment:
Would it be better to instead change the constant in the map component to be `edited`?

Is there anywhere else that uses `in progress` as the status?

I meant to clean this up when I was doing the `joinLines()` stuff.

There's a rogue use of `edited` in that method: https://github.com/developmentseed/skynet-scrub/blob/master/app/scripts/components/map/index.js#L347